metron-deployment/packer-build/README.md

```diff
@@ -0,0 +1,47 @@
+Build Metron Images
+=========================
+
+Based on the fantastic [Bento](https://github.com/chef/bento) project developed by Chef.
+
+Images Provided
+---------------------
+- hdp-centos-6.7: Centos 6.7 + HDP. Used in the quick-dev-platform Vagrant image
+- metron-centos-6.7: Centos 6.7 + HDP + Metron. Used for the codelab-platform Vagrant image.
+
+Prerequisites
+---------------------
+- Packer 0.10.1 https://www.packer.io/
+- Virtualbox 5.0.16 https://www.virtualbox.org/
+
+Build Both Images
+---------------------- 
+  Navigate to <your-project-directory>/metron-deployment/packer-build
+  Execute bin/bento
```

Comment:
Seems like I need to pass an argument to `bin/bento`?  Maybe build?  Should update the README.

```
nallen @ ~/tmp/incubator-metron-pr-123/metron-deployment/packer-build
$ ./bin/bento
Usage: bento [SUBCOMMAND [options]]

build     :   build one or more templates
help      :   prints this help message
list      :   list all templates in project
normalize :   normalize one or more templates
```